Added Content Comparision & Transfer Now sections (#3948)
umbraco/UmbracoDocs: Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

versionTo: 9.0.0
---

> # Creating, Saving and Publishing, Comparing, and Transferring Content

Reviewer: The title becomes a little cluttered. Can it somehow be made a little more simple?

Author: Changed it to Creating, Saving and Publishing Content Options

> In this section, you will get an overview of how to create a new page, save your content, options available for publishing and unpublishing the content, compare content between environments, and transfer the content from one environment to another.

Reviewer: ```suggestion
In this section, you will get an overview of how to create and save pages. You will also learn more about how to publish and unpublishing your content, as well as how you can compare content between multiple environments. Finally, you will also get an introduction to how you can transfer your created content to another environment.
```

    ![Schedualed unpublishing.](images/Schedule_Unpublishing_v9.png)
5. Select **Schedule**.

## Comparing Content between environments

:::note

> Content comparison is currently available in all Umbraco Cloud projects running Umbraco Deploy 9.3+ for Umbraco 9 and Umbraco Deploy 4.5+ for Umbraco 8.

Reviewer: Try to avoid using words like "currently", as it makes it sounds like this will be changed at some point.

Also, is is necessary to mention to specific version numbers of Deploy?
Remember that this document is intended for Content Editors, and they will usually have nothing to do with versionng.

:::

> Compare Content allows to preview content changes before transferring them to another environment. This is helpful to ensure that the correct updates are transferred when working with content in multiple environments.

Reviewer: ```suggestion
Compare Content allows previewing content changes before transferring them to another environment. This is helpful to ensure that the correct updates are transferred when working with content in multiple environments.
```

> You can see the **Summary Information** and **Field Comparision** values to understand what will change if you proceed to transfer the content upstream or try restoring content to the current environment.

Reviewer: ```suggestion
You can see the **Summary Information** and **Field Comparison** values to understand what will change if you proceed to transfer the content upstream or try restoring content to the current environment.
```

Author: Yes, I noticed the spelling is wrong, but that's how it's spelled in the UI and I kept the same as the image. Can we raise an issue to correct it?

Reviewer: Ah, I see 🤔 

Yes. You can raise an issue here: https://github.com/umbraco/Umbraco-CMS/issues

Author: Raised issue: https://github.com/umbraco/Umbraco-CMS/issues/12352

To compare content between environments, follow these steps:

1. Navigate to the page you want to compare.
2. Select the arrow next to the **Save and Publish** button. Alternatively, you can right-click the content node or click the **Actions** drop-down.

> 3. Select **Compare**.

Reviewer: ```suggestion
3. Select **Compare** to open the **Compare** window.
```

    ![Compare option](images/Compare_option.png)

> 4. In the **Compare** window, you can do the following:
>     a. **Choose the workspace** from the drop-down field.
>     b. View the **Summary information**.
>     c. In the **Field Comparision** table, view the differences between the versions in the two workspaces at the node level of each field.
>     d. Proceed to transfer the content using the **Queue for transfer** or **Transfer now** options.
>     e. Restore the content from the upstream environment using the **Partial restore** option.
>     f. Click **Close** to continue editing the content node.

Reviewer: Suggestion:
Instead of making a list within the list, wouldn't it make sense to pull the items out like this?

```suggestion
4.  **Choose the workspace** from the drop-down field.
5.  View the **Summary information**.
5. . In the **Field Comparision** table, view the differences between the versions in the two workspaces at the node level of each field.
6. Proceed to transfer the content using the **Queue for transfer** or **Transfer now** options.
7. Restore the content from the upstream environment using the **Partial restore** option.
8. Click **Close** to continue editing the content node.
```

    ![Comparing Content](images/Comparing_Content.png)

> ## Transferring content

Reviewer: I think it would be good to add a note here as well, about how this is only possible with Umbraco Cloud / Umbraco Deploy.

> You can transfer a specific content node directly to the upstream environment without adding it to the **Queue for transfer**.

Reviewer: Using words like "upstream environment" might become a bit too technical for the intended audience of this article 🙈

Author: Changed upstream to higher
